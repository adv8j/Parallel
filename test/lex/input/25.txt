int X_yacc=5;  // correct
int y=7;
int y___=8;  // correct
                     int uttaM=21;   // error
long try=9;  

   int a=d7vait =7; 
int ad=7vait=7;
long dik.psd = 17; 
    int a ='a' ; 
    
bool v =true;
float k=1.78;