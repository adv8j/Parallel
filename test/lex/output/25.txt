INT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
single line comment
INT
SHARED_IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
INT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
single line comment
INT
IDENTIFIER
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
single line comment
LONG
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
INT
IDENTIFIER
ASSIGN
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
INT
IDENTIFIER
ASSIGN
SHARED_IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
LONG
IDENTIFIER
DOT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
INT
IDENTIFIER
ASSIGN
CHARACTER
SEMICOLON
BOOL
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
FLOAT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
