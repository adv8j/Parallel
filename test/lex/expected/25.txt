INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
single_line_comment
INT
SHARED_IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
single_line_comment
INT
IDENTIFIER
Syntax Error!!!
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
single_line_comment
LONG
IDENTIFIER
Syntax Error!!!
Syntax Error!!!
ASSIGN
INT_LITERAL
SEMICOLON
INT
IDENTIFIER
ASSIGN
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
LONG
IDENTIFIER
DOT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
INT
IDENTIFIER
ASSIGN
CHARACTER_LITERAL
SEMICOLON
BOOL
IDENTIFIER
ASSIGN
TRUE
SEMICOLON
FLOAT
IDENTIFIER
ASSIGN
FLOAT_LITERAL
SEMICOLON
